flask_dance/consumer/oauth2.py

```diff
@@ -254,6 +278,18 @@ def authorized(self):
         self.session._state = state
         del flask.session[state_key]
 
+        if self._use_pkce and state:
+            code_verifier_key = f"{self.name}_{state}_oauth_code_verifier"
+            if code_verifier_key not in flask.session:
+                # can't find code_verifier, so redirect back to login view
+                log.info("code_verifier not found, redirecting user to login")
+                return redirect(url_for(".login"))
+
+            code_verifier = flask.session[code_verifier_key]
+            log.debug("code_verifier = %s", code_verifier)
+            del flask.session[code_verifier_key]
+            self.token_url_params.update({"code_verifier": code_verifier})
```

Comment:
You don't need to use `.update()` with only one value -- it's needlessly complex.
```suggestion
            self.token_url_params["code_verifier"] = code_verifier
```

Reply:
I agree, I will simplify it.